src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double high; //Add random parameters of Figure
+    private final double firstSide;
+    private final double secondSide;
+
+    private final Color color;
+
+    public IsoscelesTrapezoid(Color color, double high, double firstSide, double secondSide) {
+        this.color = color;
+        this.high = high;
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double calculateArea() {
+        return ((firstSide + secondSide) * high) / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isoscelesTrapezoid, area: "
```

Comment:
Don't use 'sq. units' in the `draw()` method's output message. Instead, just use 'sq.units' without a space to maintain consistency and avoid confusion.